ProductsService/ProductsService.Persistence/Repositories/ProductCommandRepository.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MongoDB.Driver;
+using MongoDB.Driver.Linq;
+using ProductsService.Persistence.Interfaces;
+using ProductsService.Domain.Filters;
+using ProductsService.Domain.Models;
+using ProductsService.Persistence.Specifications;
+using ProductsService.Domain.Interfaces;
+
+namespace ProductsService.Persistence.Repositories
+{
+    public class ProductCommandRepository(IMongoCommandContext context) : GenericRepository<Product>(context), IProductCommandRepository
+    {
+        public async Task<List<Product>> GetByUserIdAndByFiltersAsync(GetUsersProductsFilters filters, CancellationToken cancellationToken = default)
```

Comment:
I need to have update methods in the read repository to update the read database. And I need to have read methods in the update repository because the two database instances may be inconsistent. I can create two interfaces for the read repository. One with update methods and one without